### ISSUE TITLE
Introduce default values for spectral model parameters?

### ISSUE THREAD

**Issue by adonath** [gammapy/gammapy#992]
After working with the `FluxPointFitter` class I think it would be useful to have default values for spectral models. Instantiating the parameters with units is a non-negligible amount of typing (especially for interactive work...), while being 80% the same for most users.

In most cases you just want to fit a spectral model and only change the starting parameters if the fit fails.
In other use cases e.g. `compute_flux_points_dnde()` or `make_exposure_cube()` a spectral model is required, but only the shape of the model is important and it's somehow weird to force the user to make a choice on the `amplitude` parameter, even if it is not used at all.

Astropy as well as Sherpa have default values for their models and as the scope of Gammapy is much more limited (GeV - TeV astronomy) I think we can even make much more reasonable choices for the 
values of the parameters as e.g. Astropy does. 

In short my suggestion would be to set the default values for the all spectral models to a power law with
`index = 2.3 * u.Unit('')`, `amplitude = 1E-12 * u.Unit('cm-2 s-1 TeV-1')` and `reference = 1. * u.TeV`. The only tricky case is `ExponentialCutoffPowerLaw3FGL`, where a `ecut` must be set to a very large value.

Any opinion on this @cdeil, @joleroi, @jjlk, @leajouvin?

**Comment by cdeil** [comment]
I'm 50:50 on this suggestion. As you noted already, there aren't good defaults for all parameters, so whatever we put will mean that some users will get bitten. On the other hand it's convenient to have defaults here sometimes.

PS: I like passing strings for quantities in examples / docs, e.g.
```python
pl = PowerLaw(amplitude='1e-12 cm-2 s-1 TeV-1', index=2, reference='1 TeV')
```
which works if inputs are passed through constructors like `Quantity` (since Astropy 1.3 which we require now in Gammapy) and `Angle`. I think if we had that, the desire for defaults would be reduced (but we might still add them).

**Comment by adonath** [comment]
Default parameters have been introduced in #1090 and creating a model instance with parameter specification as string also works (however it should be maybe documented...). I am closing this issue now.
